package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@groveco/backbone.store",
   "main": "dist/index.js",
-  "version": "0.3.0",
+  "version": "0.3.1-1",
```

Comment:
If we merge #52, it's not necessary to bump the version on every build.

Reply:
That's good to know. I'll leave it at `0.3.1` ?